gridfinity-rebuilt-baseplate.scad

```diff
@@ -213,6 +216,11 @@ module gridfinityBaseplate(grid_size_bases, length, min_size_mm, sp, hole_option
             translate([0, 0, additional_height/2])
             cutter_screw_together(grid_size.x, grid_size.y, length);
         }
+        
+        if (st == 1) {
+            translate([0, 0, baseplate_height_mm])
+            cube([10000, 10000, LAYER_HEIGHT*4], center=true);
```

Comment:
Where possible, magic numbers should be avoided.

Why 10000, instead of the breastplate size?  Why 4?

Reply:
Replaced the 10000 by a parametric value even though I do not think anyone will ever generate base plates larger than 10×10m ;)

4 is derived from the cube being centered and us needing to cut of 2 layers from the top.  Why 2?  Because this is the necessarry flattening required to allow 2 perimeters to be printed side-by-side on the topmost layer.